internal/mock/mock_deviceprofileclient.go

```diff
@@ -19,7 +19,7 @@ var (
 	DeviceProfileRandomUnsignedGenerator       = models.DeviceProfile{}
 	DeviceProfileRandomFloatGenerator          = models.DeviceProfile{}
 	DuplicateDeviceProfileRandomFloatGenerator = models.DeviceProfile{}
-	NewDeviceProfile 							= models.DeviceProfile{}
+	NewDeviceProfile                           = models.DeviceProfile{}
```

Comment:
whitespace only changes in this file (see above)!

Reply:
white space changed by go fmt